### DISCUSSION TITLE
求教~自定义prot文字超出隐藏怎么 有偿解决 QQ 664987537

### DISCUSSION THREAD

**Discussion by DividingLine** [antvis/X6#3396] [Q&A]
<img width="436" alt="image" src="https://user-images.githubusercontent.com/33141363/225893503-dfd1232d-4094-41a6-b929-c717a459a851.png">
![image](https://user-images.githubusercontent.com/33141363/225893684-d2a07684-6893-423e-a859-5585edc17c00.png)

**Comment by DividingLine** [comment]
![image](https://user-images.githubusercontent.com/33141363/225893844-cc267f0a-4457-40cb-bb8b-5e19fa75b7db.png)

**Reply by NewByVector** [reply]
使用 [textWrap](https://x6.antv.antgroup.com/api/registry/attr#textwrap) 属性。

**Reply by DividingLine** [reply]
不行啊  是给哪个属性加？

**Reply by Epic-Deno** [reply]
我加你了
